### DISCUSSION TITLE
Examples for closed loop

### DISCUSSION THREAD

**Discussion by younesStrittmatter** [AutoResearch/autora#375] [Show and Tell]
The restructuring of the project made it very easy for me to publish a runner package, and by using John's autora-workflow package (Huge shoutout to @hollandjg), I could get two closed loops running. Here are the examples: [autora-closed-loop-firebase-tutorial](https://github.com/AutoResearch/autora-closed-loop-firebase-tutorial). 
This is supposed to be a tutorial on how to set it up, so if somebody wants to try, feel free. It worked for me, and I have addressed some issues from the team meeting in the documentation.
I didn't include the prolific functionality in the tutorial yet (recruiting real participants from prolific), but there is a finished firebase_prolific_runner ready to be tested.

**Comment by musslick** [comment]
Awesome to hear, would love to give that a try!

**Reply by younesStrittmatter** [reply]
The README on the main page of the repository has tutorials on how to set up and autora workflow and how to set up firebase. And then, one can follow along with the specific examples on how to use jsPsych, super-experiment or seetBean. I think jsPsych is a good starting point.
